packages/core/src/Provider/HandleProvider/types.ts

```diff
@@ -0,0 +1,28 @@
+import { ByronAddress, EnterpriseAddress, PaymentAddress, PointerAddress } from '../../Cardano/Address';
+import { Point } from '../..';
+
+export type Handle = string;
+
+export type CardanoAddress = ByronAddress | PaymentAddress | EnterpriseAddress | PointerAddress;
+
+export enum HandleIssuer {
+  KoraLabs = 'KoraLabs'
+}
+export interface HandleResolution {
+  issuer: HandleIssuer;
```

Comment:
```suggestion
  policyId: Cardano.PolicyId;
```
Thoughts on this @mkazlauskas? The extra construct is unnecessary and makes our implementation static when all we need here the policy ID to differentiate the issuer.

Reply:
Good idea, agree

Reply:
So in the HandleProvider, where do I get the `policyId`? The `IHandle` API response, or the `tip`, from Cardano tip, con't have it, I believe.

Reply:
It's static for cora labs provider: you could either determine network (e.g. preprod/mainnet) by URL, or just add a new `policyId` arg to the constructor

Reply:
> or just add a new policyId arg to the constructor

This. Best to keep the knowledge of the actual `policyId` at the configuration level